src/app/services/auth.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+import { IUser } from '../models/user-model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  public user = new Subject<IUser>();
+
+  public getUser(value: IUser) {
```

Comment:
next is for setting value into subject. so getUser is incorrect naming for this case, use setUser instead

Reply:
done